modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -388,27 +388,30 @@
 			if(!forced)
 				to_chat(user, span_notice("Some careful placement and stoking of the flame will allow you to keep at least the embers burning..."))
 			minimum_target_temperature = 25 // Will allow quicker reheating from having no fuel
-			temperature_loss_reduction = 2
+			temperature_loss_reduction = 3
```

Comment:
the temp changes are undocumented, i get why you're doing this but please add it to the PR and CL